pkgs/tools/networking/dublin-traceroute/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, jsoncpp, libtins, libpcap, openssl, nixosTests }:
+
+stdenv.mkDerivation {
+  pname = "dublin-traceroute";
+  version = "unstable-2023-04-12";
+
+  src = fetchFromGitHub {
+    owner = "insomniacslk";
+    repo = "dublin-traceroute";
+    rev = "2fb78ea05596dfdf8f7764b497eb8d3a812cb695";
+    hash = "sha256-E1HYMd0wDTfAZ0TamQFazh8CPhMa2lNIbF4aEBf5qhk=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [ jsoncpp libtins libpcap openssl ];
+
+  outputs = [
+    "out"
+    "lib"
+    "dev"
+  ];
+
+  meta = with lib; {
+    description = "NAT-aware multipath traceroute tool";
+    homepage = "https://dublin-traceroute.net/";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ baloo ];
+    platforms = platforms.unix;
+    mainProgram = "dublin-traceroute";
+  };
+
+  passthru.tests = {
+    inherit (nixosTests) dublin-traceroute;
+  };
```

Comment:
I usually silence the `attribute-ordering` warnings in nixpkgs-hammering, but `meta` is strictly preferred to be the last attribute.
```
warning: attribute-ordering
The attribute “passthru” should preferably come before “meta” attribute in the expression.

Near /dev/shm/nixpkgs-review/pr-261704/nixpkgs/pkgs/tools/networking/dublin-traceroute/default.nix:24:3:
   |
24 |   meta = with lib; {
   |   ^
Near /dev/shm/nixpkgs-review/pr-261704/nixpkgs/pkgs/tools/networking/dublin-traceroute/default.nix:33:3:
   |
33 |   passthru.tests = {
   |   ^
See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md
```